cmd/neofs-lens/internal/storage/list.go

```diff
@@ -0,0 +1,48 @@
+package storage
+
+import (
+	"errors"
+	"io"
+
+	common "github.com/nspcc-dev/neofs-node/cmd/neofs-lens/internal"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/engine"
+	"github.com/spf13/cobra"
+)
+
+var storageListObjsCMD = &cobra.Command{
+	Use:   "list",
+	Short: "Object listing",
+	Long:  `List all objects stored in a blobstor (as registered in metabase).`,
+	Args:  cobra.NoArgs,
+	Run:   listFunc,
+}
+
+func init() {
+	common.AddConfigFileFlag(storageListObjsCMD, &vConfig)
+}
+
+func listFunc(cmd *cobra.Command, _ []string) {
+	// other targets can be supported
+	w := cmd.OutOrStderr()
+
+	storage := openEngine(cmd)
+	defer storage.Close()
+
+	var p engine.ListWithCursorPrm
+	p.WithCount(1024)
```

Comment:
magic? can lose 1025-th object and do not understand it?

Reply:
It will list everything. Yeah, magic.

Reply:
oh, almost endless `for`, did not see it. check context b/w iteration then to make it controllable?